kqueen/engines/openstack_kubespray.py

```diff
@@ -64,10 +64,20 @@ class OpenstackKubesprayEngine(BaseEngine):
                     "required": True,
                 },
             },
-            "flavor": {
+            "master_flavor": {
                 "type": "text",
                 "order": 30,
-                "label": "Flavor",
+                "label": "Master flavor",
+                "default": config.get("KS_MASTER_FLAVOR"),
```

Comment:
imo its better to set defaults directly here, for ex m1.medium for master, m1.large for slaves

Reply:
decided not to use values from config at all. may be later use profile for each provisioner with predefined settings